network_tools/test/host_scanner_test.dart

```diff
@@ -51,24 +51,47 @@ void main() {
       );
     });
 
+    //todo: this test is not working on windows, not matter what.
     test('Running scanDevicesForSinglePort tests', () {
       expectLater(
         HostScanner.scanDevicesForSinglePort(
           interfaceIp, port, //ssh should be running at least in any host
```

Comment:
I don't think that SSH can be found on all machines.
Maybe this is why the test on Windows fails.

If you want to test that there is at least one port open add a function that opens a port.

https://blog.smallsec.ca/default-open-ports-in-windows/

Some windows does not have any port open.